ocaml/testsuite/tests/language-extensions/pprintast_unconditional.ml

```diff
@@ -20,11 +20,13 @@ module Example = struct
 
   let modality_record  = parse module_expr
     "struct \
-      type t = {global_ x : string; global_ y : int} \
```

Comment:
Can you add a test case without a legacy modality in it? All the constructor args in these examples have `global_` on them

Reply:
Good point